src/content/structured/components/checkboxes/code.mdx

```diff
@@ -140,66 +144,48 @@ export const snippetsHelperText = [
-  <IcCheckbox value="valueName1" label="Unselected / Small" />
-  <IcCheckbox value="valueName2" label="Selected / Small" checked />
+    snippet: `<IcCheckboxGroup label="Make changes to your order" name="1" size="small">
+  <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+  <IcCheckbox value="valueName2" label="Soya milk" checked />
 </IcCheckboxGroup>
-<IcCheckboxGroup label="This is a default checkbox group" name="2">
-  <IcCheckbox value="valueName1" label="Unselected / Default" />
-  <IcCheckbox value="valueName2" label="Selected / Default" checked />
+<IcCheckboxGroup label="Make changes to your order" name="2">
+  <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+  <IcCheckbox value="valueName2" label="Soya milk" checked />
 </IcCheckboxGroup>
-<IcCheckboxGroup label="This is a large checkbox group" name="3" size="large">
-  <IcCheckbox value="valueName1" label="Unselected / Large" />
-  <IcCheckbox value="valueName2" label="Selected / Large" checked />
-</IcCheckboxGroup>
-<IcCheckboxGroup label="Checkboxes define size in this checkbox group" name="4">
-  <IcCheckbox value="valueName1" label="Small" size="small" />
-  <IcCheckbox value="valueName2" label="Default" />
-  <IcCheckbox value="valueName3" label="Large" size="large" />
+<IcCheckboxGroup label="Make changes to your order" name="3" size="large">
+  <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+  <IcCheckbox value="valueName2" label="Soya milk" checked />
 </IcCheckboxGroup>`,
   },
 ];
 
 <ComponentPreview snippets={snippetsSizes} style={{ gap: "24px" }}>
-  <IcCheckboxGroup label="This is a small checkbox group" name="1" size="small">
-    <IcCheckbox value="valueName1" label="Unselected / Small" />
-    <IcCheckbox value="valueName2" label="Selected / Small" checked />
-  </IcCheckboxGroup>
-  <IcCheckboxGroup label="This is a default checkbox group" name="2">
-    <IcCheckbox value="valueName1" label="Unselected / Default" />
-    <IcCheckbox value="valueName2" label="Selected / Default" checked />
+  <IcCheckboxGroup label="Make changes to your order" name="1" size="small">
+    <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+    <IcCheckbox value="valueName2" label="Soya milk" checked />
   </IcCheckboxGroup>
-  <IcCheckboxGroup label="This is a large checkbox group" name="3" size="large">
-    <IcCheckbox value="valueName1" label="Unselected / Large" />
-    <IcCheckbox value="valueName2" label="Selected / Large" checked />
+  <IcCheckboxGroup label="Make changes to your order" name="2">
+    <IcCheckbox value="valueName1" label="Extra shot (50p)" />
+    <IcCheckbox value="valueName2" label="Soya milk" checked />
   </IcCheckboxGroup>
-  <IcCheckboxGroup
-    label="Checkboxes define size in this checkbox group"
-    name="4"
-  >
-    <IcCheckbox value="valueName1" label="Small" size="small" />
-    <IcCheckbox value="valueName2" label="Default" />
-    <IcCheckbox value="valueName3" label="Large" size="large" />
```

Comment:
Think we need to bring back this code example as it shows setting each checkbox size rather than the whole checkbox group. Might want to add it back as a completely separate component preview so it's obvious.

Reply:
Done but will need help with creating a new snippet.